spark/dl/src/main/scala/org/apache/spark/ml/DLEstimator.scala

```diff
@@ -67,68 +166,61 @@ class DLEstimator[@specialized(Float, Double) T: ClassTag](
 
   def setBatchSize(value: Int): this.type = set(batchSize, value)
 
-  /**
-   * optimization method to be used. BigDL supports many optimization methods like Adam,
-   * SGD and LBFGS. Refer to package com.intel.analytics.bigdl.optim for all the options.
-   * Default: SGD
-   */
-  val optimMethod = new Param[OptimMethod[_]](this, "optimMethod", "optimMethod")
-
-  def getOptimMethod: OptimMethod[_] = $(optimMethod)
-
-  def setOptimMethod(value: OptimMethod[_]): this.type = set(optimMethod, value)
-
-  /**
-   * number of max Epoch for the training, an epoch refers to a traverse over the training data
-   * Default: 100
-   */
-  val maxEpoch = new IntParam(this, "maxEpoch", "number of max Epoch", ParamValidators.gt(0))
-  setDefault(maxEpoch -> 100)
-
-  def getMaxEpoch: Int = $(maxEpoch)
+  def setOptimMethod(value: OptimMethod[T]): this.type = set(optimMethod, value)
 
   def setMaxEpoch(value: Int): this.type = set(maxEpoch, value)
-
-  /**
-   * learning rate for the optimizer in the DLEstimator.
-   * Default: 0.001
-   */
-  val learningRate = new DoubleParam(this, "learningRate", "learningRate", ParamValidators.gt(0))
-
-  setDefault(learningRate -> 1e-3)
-
-
-  def getLearningRate: Double = $(learningRate)
+  setDefault(maxEpoch -> 100)
```

Comment:
100 epoch can be too large?

Reply:
Just want to ensure that we got converged model on some smaller data set. 
I can change it to 50.